presets/4.3/rc_link/elrs_250hz.txt

```diff
@@ -0,0 +1,23 @@
+# Title: ELRS 250hz
+# FirmwareVersion: 4.3
+# Category: RC_LINK
+# Official: true
+# Keywords: ELRS, rc, link, 250hz
+# Author: ctzsnooze
+# Description: Basic RC link settings for a 250hz ELRS link via CRSF
+# Description: WARNING: make ABSOLUTELY SURE that the Hardware > ADC Filter is un-checked!
+# Description: A compatible version of EdgeTx or OpenTx is required
+# Description: The settings are intended for race or fast freestyle flying
+# Description: For smooth cinematic or HD flying, increase smooth_factor to 60
```

Comment:
I was thinking that we agreed that **RC_LINK** and **RC_SMOOTHING** categories should have completely separate sets of CLI commands/variables?
Remember, when someone applies `ERLS 250hz Preset` + `Cinematic Preset` it should be equal to when someone applies `Cinematic Preset` + `ERLS 250hz Preset`

Reply:
resolved